test/test_functional_tensor.py

```diff
@@ -19,6 +22,21 @@ def test_hflip(self):
         self.assertEqual(hflipped_img.shape, img_tensor.shape)
         self.assertTrue(torch.equal(img_tensor, hflipped_img_again))
 
+    def test_crop(self):
+        img_tensor = torch.FloatTensor(3, 16, 16).uniform_(0, 1)
```

Comment:
Can you use `torch.rand(3, 16, 16)` instead? `torch.FloatTensor` is a legacy API and is deprecated